site/content/docs/5.2/components/dropdowns.md

```diff
@@ -230,7 +230,12 @@ And putting it to use in a navbar:
 {{< example >}}
 <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
   <div class="container-fluid">
-    <a class="navbar-brand" href="#">Navbar</a>
+    <div class="navbar-brand me-auto me-lg-4">
```

Comment:
In order to be consistent with the other pages, we shouldn't use here `me-auto me-lg-4` and rather make the modification everywhere in another PR.

```suggestion
    <div class="navbar-brand">
```